src/components/navigation/z-menu/index.tsx

```diff
@@ -157,31 +175,32 @@ export class ZMenu {
     return (
       <Host>
         {this.renderMenuLabel()}
-        <div
-          class="content"
-          ref={(el) => {
-            this.content = el;
-          }}
-          hidden={!this.open}
-        >
-          {this.hasHeader && (
-            <header class="header">
-              <slot
-                name="header"
-                onSlotchange={this.checkContent}
-              ></slot>
-            </header>
-          )}
+
+        {this.hasContent && (
           <div
-            class="items"
-            role="menu"
+            class="content"
+            ref={(el) => (this.content = el)}
           >
-            <slot
-              name="item"
-              onSlotchange={this.onItemsChange}
-            ></slot>
+            {this.hasHeader && (
+              <header class="header">
+                <slot
+                  name="header"
+                  onSlotchange={this.checkContent}
```

Comment:
```suggestion
                  onSlotchange={this.checkContent.bind(this)}
```

Reply:
already explained [here](https://github.com/ZanichelliEditore/design-system/pull/339#discussion_r1138343172) why I didn't put the `bind`